drivers/dac/ltc268x/ltc268x.h

```diff
@@ -91,7 +91,7 @@
 #define LTC268X_DITH_EN(x)			(1 << ((x) & 0xF))
 
 /* Number of channels corresponding to device */
-const uint8_t ltc268x_channels[] = { 8, 16};
+extern const uint8_t ltc268x_channels[];
```

Comment:
is this array going to be accessed outside of the driver? otherwise I would remove the external declaration and simply define it statically in ltc268x.c.

Reply:
I think this would be useful for the application layer to get number of channels for the desired part number.

Reply:
Then I would suggest adding a new field uint8_t num_channels in struct ltc268x_dev, which is populated at init time and is accessible to the application layer, and then use that field instead of the array in the driver.